package calc;

import org.junit.Rule;
import org.junit.Test;
import org.junit.internal.runners.statements.ExpectException;

public class ExceptionTest {


    public void testException (){

//        @Test (expected = DivisionException.class)
//                public void testException()
//                Division division  = new Division(1, 0);
//        division.calculateResult();
//
//        @Rule
//        public ExpectException exception = ExpectException.none();

    }
}
